buildenv/jenkins/jobs/pull-requests/PullRequest-zos_390-64

```diff
@@ -7,7 +7,7 @@ pipeline {
         stage('Get Sources') {
             steps {
                 timestamps {
-                    checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '${sha1}']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[name: 'origin', refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: 'git://github.com/eclipse/omr.git']]]
+                    checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '${sha1}']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[name: 'origin', refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: 'git@github.com:eclipse/omr.git']]]
```

Comment:
I also just realized the file you're modifying is no longer in use in our builds. The only url affecting this is configured in each job. I think at the time we added the new file, Jan requested we leave the old files intact as he was using them on his farm.